auth_backends/backends.py

```diff
@@ -34,6 +30,22 @@ class EdXOpenIdConnect(OpenIdConnectAuth):
 
     auth_complete_signal = django.dispatch.Signal(providing_args=["user", "id_token"])
 
+    @property
+    def ID_TOKEN_ISSUER(self):
+        return self.setting('ISSUER')
```

Comment:
It seems this advice was never heeded, and now I have one more thing to fix for LEARNER-693. Come on!